sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/AggregateOptimizeSuite.scala

```diff
@@ -29,7 +29,8 @@ import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.internal.SQLConf.{CASE_SENSITIVE, GROUP_BY_ORDINAL}
 
 class AggregateOptimizeSuite extends PlanTest {
-  override val conf = new SQLConf().copy(CASE_SENSITIVE -> false, GROUP_BY_ORDINAL -> false)
+  SQLConf.get.setConf(CASE_SENSITIVE, false)
+  SQLConf.get.setConf(GROUP_BY_ORDINAL, false)
```

Comment:
Can we follow https://github.com/apache/spark/pull/30097/files#diff-6f79a980de03203104cfacc6f50c302b843e3b60f1d48942bded6706c75a02b7R128 , instead of updating the global conf object?

Reply:
done